src/styles/_variables.scss

```diff
@@ -0,0 +1,21 @@
+// Colors
+$color-primary: #4a9dec;
+$color-background: #f3f3f3;
+$color-background-hover: white;
+$color-border-transparent: transparent;
+
+// Sizes
+$size-border-width: 2px;
+$size-border-radius: 10px;
+$size-input-height: 40px;
+$size-spacing-sm: 8px;
+$size-spacing-md: 12px;
```

Comment:
`size-` is unnecessary. Can we just make these like `input_height`, `spacing-md`, `border-radius`?